app/src/utils/services/DataService.ts

```diff
@@ -46,11 +46,16 @@ class DataService {
                                 location: Location|undefined,
                                 radius: string
                             }): Promise<any> {
-        const query = new QueryBuilder();
-        searchValues.forEach((value) => {
-            query.shouldMatch('categories', value)
-                .shouldMatch('title', value);
-        });
+        const query: QueryBuilder = new QueryBuilder();
+        console.log(searchValues);
+        if (searchValues.join(',') !== '') {
```

Comment:
I think if we pass an array of empty strings in we have a problem before this function gets called and should be solved there. @mxxbe how do you see this?